### PR TITLE
Sprint2 - Session Configuration Menu

### PR DESCRIPTION
**Pull request by Ay1tsMe**
Ready to merge. Web-page is accessible through `/session` so test it out!!!!

I added the `Semester` input field because that's what he had in the demo. We could probably use the current date to work out which semester it is but leave it for now. 

Also there is a clock that displays the `Current Time` for the user. Not sure if this is necessary but their were discussions about whether we want this instead of "Morning" or "Afternoon" options.

### CODE REVIEW

**Comment by livvy10** [comment]
Looks really good so far, I liked your code style, it's written very clearly. I accessed the session page as you said and filled in the form successfully!
Considering this is for configuring a session by facilitators, I believe Adrian doesn't want them to have a semester choice as that will already known by the db.
I think instead of having text boxes for inputs where an error is shown if it is invalid, there should just be selection boxes and you have to pick from one of the valid units. (Although keep html validation in the unlikely instance they muck around with the actual html and try to submit an invalid input - I think that's a thing??)
Also I think this should be done in a form with forms.py - not sure if this was something to change to at a later stage though.

File: app/templates/base.html [hunk 1]
```diff
@@ -14,7 +14,12 @@
 	<script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"
 		integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz"
 		crossorigin="anonymous"></script>
+
+	<!-- Linking main.js -->
+    <script src="{{ url_for('static', filename='js/main.js') }}"></script>
+	<!-- Linking main.css -->
+    <script src="{{ url_for('static', filename='styles/main.css') }}"></script>
```
Comment (DeltaO3) [inline app/templates/base.html:21]: Pretty sure we want to use <link> to include our css, and CSS should be in the head, after the bootstrap link.
The script included on line 14 is a CDN for jquery (and thus a script). not 100% certian here, but fairly certain.

File: app/templates/session.html [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+        <input type="hidden" id="session_time" name="session_time">
+    </div>
+
+    <!-- Display Generated Database Name -->
+    <div>
+        <label for="database_name_display">Database Name:</label>
+        <div id="database_name_display"></div>
+    </div>
+
+    <!-- Buttons -->
```
Comment (DeltaO3) [inline app/templates/session.html:55]: Buttons should include the class btn, but i understand you've just gone for the html/js skeleton here, so its fine for now.

**Review by DeltaO3 — CHANGES_REQUESTED** [review]
Overall looks pretty good, skeleton functionality is great!
Main thing is the client said he didn't want semester, spent a while going over that point last meeting. I think we should just not include it (for now) so that if we show him this next meeting he doesn't think we didnt listen to him.

Minor comments below on things i'd change but am fine without for now, due to the skeleton nature of the html file.

**Comment by Ay1tsMe** [comment]
Fixed all the issues brought up. Should be good to go

**Review by DeltaO3 — APPROVED** [review]
LGTM! Just be wary when updating from main - you have some files that were manually merged from a branch but are now merged, so you aren't up to date. Hopefully git is smart.

**Review by livvy10 — APPROVED** [review]
All the new changes look great!
In the future I think the unit selected should determine which sessions can be selected from - but this is beyond the scope of this issue.

**Pull request merged**